packages/main/src/Slider.js

```diff
@@ -0,0 +1,457 @@
+
+// Template
+import SliderTemplate from "./generated/templates/SliderTemplate.lit.js";
+
+// Styles
+import SliderStyles from "./generated/themes/Slider.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-slider",
+	altTag: "ui5-slider",
+	languageAware: true,
+	managedSlots: true,
+	properties: /** @lends sap.ui.webcomponents.main.Slider.prototype */  {
+		/**
+		 * Minimum value of the slider
+		 * <br><br>
+		 *
+		 * @type {Float}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		min: {
+			type: Float,
+		},
+		/**
+		 * Maximum value of the slider
+		 * <br><br>
+		 *
+		 * @type {Float}
+		 * @defaultvalue 100
+		 * @public
+		 */
+		max: {
+			type: Float,
+		},
+		/**
+		 * Defines the size of the slider's selection intervals. (e.g. min = 0, max = 10, step = 5 would result in possible selection of the values 0, 5, 10).
+		 * When 0 or a negative number, the component fallbacks to its default value.
+		 * <br><br>
+		 *
+		 * @type {Integer}
+		 * @defaultvalue 1
+		 * @public
+		 */
+		step: {
+			type: Float,
+			defaultValue: 1,
+		},
+		/**
+		 * Current value of the slider
+		 * <br><br>
+		 *
+		 * @type {Float}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		value: {
+			type: Float,
+			defaultValue: 0,
+		},
+		/**
+		 * Determines end point of a selection - position of a second handle on the slider. Allow defining of a <b>selected range</b>.
+		 * When <code>endValue</code> is specified, the <code>value</code> property is used as a "start value" to determine a selected range on the slider.
+		 * If the values is lower/higher than the allowed minimum/maximum, they will be set to the corresponding min/max values of the slider.
+		 * <br><br>
+		 *
+		 * @type {Float}
+		 * @defaultvalue 0
+		 * @public
+		 */
+		 endValue: {
+			type: Float,
+		},
+		/**
+		 * Enables tick marks visualization for each step. The step value must not be set to 0
+		 * <br><br>
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		 tickmarks: {
+			type: Boolean,
+		},
+		/**
+		 * Defines whether the <code>ui5-slider</code> is in disabled state.
+		 * <br><br>
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+	},
+	slots: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Defines the text of the <code>ui5-range-slider</code>.
+		 * <br><b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
+		 *
+		 * @type {Node[]}
+		 * @slot
+		 * @public
+		 */
+		"default": {
+			type: Node,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Fired when the value changes and the user has finished interacting with the slider.
+		 *
+		 * @event
+		 * @public
+		*/
+		change: {},
+		/**
+		 * Fired when the value changes due to user interaction.
```

Comment:
You have to clarify that it is fired  "during the dragging period" otherwise both sound too similar